.github/workflows/pythonapp.yml

```diff
@@ -73,7 +73,7 @@ jobs:
         pip install pandas
     - uses: actions/cache@v1
       # TVM takes forever, we try to cache it.
-      if: ${{ matrix.python-version != '3.5' && matrix.os != 'windows-2019' }}
+      if: ${{ matrix.python-version != '3.6' && matrix.os != 'windows-2019' }}
```

Comment:
Since we removed 3.5, I don't think we need to have special paths for 3.6 and 3.7. We can just use 3.7. This should also speed up the pipeline hopefully

Reply:
ok removing that part!